src/utils/tests/nviHelpers.test.ts

```diff
@@ -0,0 +1,47 @@
+import { describe, test } from 'vitest';
+
+describe('willResetNviStatuses()', () => {
+  test('Returns false when result cannot be an NVI candidate', () => {
+    // TODO
+  });
+
+  test('Returns true when year is changed', () => {
+    // TODO
+  });
+
+  test('Returns false when date is changed within same year', () => {
+    // TODO
+  });
+
+  test('Returns true when category is changed', () => {
+    // TODO
+  });
+
+  test('Returns true when journal or publisher is changed', () => {
+    // TODO
+  });
+
+  test('Returns true when series is changed', () => {
+    // TODO
+  });
+
+  test('Returns true when a new institution is added as affiliation', () => {
```

Comment:
Maybe split this, if practical?

When modifying the affiliations of an existing contributor, we reset in two different cases:
- Adding a new affiliation with a different top-level organization
- Changing the top-level organization for the existing affiliation